src/utils/load-image.tsx

```diff
@@ -6,18 +6,22 @@ import { renderAsEntity } from "../utils/jsx-entity";
 import { HubsWorld } from "../app";
 import { Texture } from "three";
 import { AlphaMode } from "./create-image-mesh";
+import { MEDIA_LOADER_FLAGS } from "../bit-systems/media-loading";
 
-export function* loadImage(world: HubsWorld, url: string, contentType: string) {
+export function* loadImage(world: HubsWorld, flags: number, url: string, contentType: string) {
   const { texture, ratio, cacheKey }: { texture: Texture; ratio: number; cacheKey: string } =
     yield loadTextureCancellable(url, 1, contentType);
+  const projection =
+    flags & MEDIA_LOADER_FLAGS.SPHERICAL_PROJECTION ? ProjectionMode.SPHERE_EQUIRECTANGULAR : ProjectionMode.FLAT;
```

Comment:
Feels weird that this gets a MEDIA_LOADER_FLAGS instead of a PROJECTION_MODE. Feels like the conversion between those should be at the thing calling `loadImage` this would also get rid of the confusing `0` being passed in (it would become `ProjectionMode.FLAT`.